src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -803,10 +803,13 @@ public boolean configure(StaplerRequest req, JSONObject formData) throws FormExc
         
 
         public FormValidation doCheckAdminlist(@QueryParameter String value) throws ServletException {
-            if (!adminlistPattern.matcher(value).matches()) {
-                return FormValidation.error("GitHub username may only contain alphanumeric characters or dashes "
-                        + "and cannot begin with a dash. Separate them with whitespaces.");
+            for (String admin : value.split("\\s")) {
```

Comment:
This line is ultimately my concern. I believe some users will want to enter with newline characters rather than whitespace. We'll need to catch those in the form validation. So I'd want a test for that. Does that make sense?